maintainer-list.nix

```diff
@@ -194,6 +194,65 @@
     githubId = 46590321;
     name = "xrelkd";
   };
+  suhr = {
+    email = "suhr@i2pmail.org";
+    github = "suhr";
+    githubId = 65870;
+    name = "Сухарик";
+  };
+  jonafato = {
+    email = "jon@jonafato.com";
+    github = "jonafato";
+    githubId = 392720;
+    name = "Jon Banafato";
+  };
+  ralith = {
+    email = "ben.e.saunders@gmail.com";
+    github = "ralith";
+    githubId = 104558;
+    name = "Benjamin Saunders";
+  };
+  nyanloutre = {
+    email = "paul@nyanlout.re";
+    github = "nyanloutre";
+    githubId = 7677321;
+    name = "Paul Trehiou";
+  };
+  johnazoidberg = {
+    email = "git@danielschaefer.me";
+    github = "johnazoidberg";
+    githubId = 5307138;
+    name = "Daniel Schäfer";
+  };
+  chiiruno = {
+    email = "okinan@protonmail.com";
+    github = "chiiruno";
+    githubId = 30435868;
+    name = "Okina Matara";
+  };
+  tadfisher = {
+    email = "tadfisher@gmail.com";
+    github = "tadfisher";
+    githubId = 129148;
+    name = "Tad Fisher";
+  };
+  rvolosatovs = {
+    email = "rvolosatovs@riseup.net";
+    github = "rvolosatovs";
+    name = "Roman Volosatovs";
+  };
+  catern = {
+    email = "sbaugh@catern.com";
+    github = "catern";
+    githubId = 5394722;
+    name = "Spencer Baugh";
+  };
+  vdemeester = {
+    email = "vincent@sbr.pm";
+    github = "vdemeester";
+    githubId = 6508;
+    name = "Vincent Demeester";
+  };
```

Comment:
Can you delete them from the commented section as well?

Reply:
Sure, incoming shortly